uploader.json

```diff
@@ -60,6 +60,11 @@
             "pattern": [],
             "symbolic_name": "certificates_enddate"
         },
+        {
+            "command": "/usr/bin/find /sys/devices/pci0000:00 -mindepth 8 -maxdepth 8",
+            "pattern": [],
+            "symbolic_name": "pci_rport_target_disk_paths"
+        },
```

Comment:
@xhe123 can you please move this command entry so that it will be alpha sorted order by symbolic name?  Same with the change in `uploader.v2.json`.

Reply:
@bfahr I've put this spec in front of 'pcs_status' for uploader.json and uploader.v2.json. Does it look ok to you? Thanks!